test/model/content_test.dart

```diff
@@ -1,16 +1,429 @@
+      '<br>\n</p>\n</blockquote>',
+    [QuotationNode([MathBlockNode(texSource: r'\lambda')])]);
+
+  static const singleImage = ContentExample(
+    'single image',
+    "https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3",
+    '<div class="message_inline_image">'
+      '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">'
+        '<img src="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3"></a></div>', [
+    ImageNodeList([
+      ImageNode(srcUrl: 'https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3'),
+    ]),
+  ]);
+
+  static const multipleImages = ContentExample(
+    'multiple images',
+    "https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3\nhttps://chat.zulip.org/user_avatars/2/realm/icon.png?version=4",
+    '<p>'
+      '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3</a><br>\n'
+      '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4">https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4</a></p>\n'
+    '<div class="message_inline_image">'
+      '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3">'
+        '<img src="https://uploads.zulipusercontent.net/f535ba07f95b99a83aa48e44fd62bbb6c6cf6615/68747470733a2f2f636861742e7a756c69702e6f72672f757365725f617661746172732f322f7265616c6d2f69636f6e2e706e673f76657273696f6e3d33"></a></div>'
+    '<div class="message_inline_image">'
+      '<a href="https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4">'
+        '<img src="https://uploads.zulipusercontent.net/8f63bc2632a0e41be3f457d86c077e61b4a03e7e/68747470733a2f2f636861742e7a756c69702e6f72672f757365725f617661746172732f322f7265616c6d2f69636f6e2e706e673f76657273696f6e3d34"></a></div>', [
+    ParagraphNode(links: null, nodes: [
+      LinkNode(url: 'https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3', nodes: [TextNode('https://chat.zulip.org/user_avatars/2/realm/icon.png?version=3')]),
+      LineBreakInlineNode(),
+      TextNode('\n'),
+      LinkNode(url: 'https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4', nodes: [TextNode('https://chat.zulip.org/user_avatars/2/realm/icon.png?version=4')]),
+    ]),
+    ImageNodeList([
+      ImageNode(srcUrl: 'https://uploads.zulipusercontent.net/f535ba07f95b99a83aa48e44fd62bbb6c6cf6615/68747470733a2f2f636861742e7a756c69702e6f72672f757365725f617661746172732f322f7265616c6d2f69636f6e2e706e673f76657273696f6e3d33'),
+      ImageNode(srcUrl: 'https://uploads.zulipusercontent.net/8f63bc2632a0e41be3f457d86c077e61b4a03e7e/68747470733a2f2f636861742e7a756c69702e6f72672f757365725f617661746172732f322f7265616c6d2f69636f6e2e706e673f76657273696f6e3d34'),
+    ]),
+  ]);
+
+  static const contentAfterImageCluster = ContentExample(
+    'content after image cluster',
```

Comment:
(But it's fine and good for these description strings to remain in more natural English syntax, like this is.)